system/jlib/jhash.hpp

```diff
@@ -49,11 +49,8 @@ interface jlib_decl IAtom : extends IMapping
 {
  public:
     virtual const char * getNamePtr() const = 0;
```

Comment:
can we rename this function to something that would be sensible to call.

e.g., queryStr()

Some of the instances that have been converted could call this directly if we know that this != NULL.